src/binary-reader-logging.cc

```diff
@@ -353,6 +353,11 @@ Result BinaryReaderLogging::OnTryExpr(Index num_types, Type* sig_types) {
   return reader_->OnTryExpr(num_types, sig_types);
 }
 
+Result  BinaryReaderLogging::OnSimd1LaneImmExpr(Opcode opcode, Type type, uint32_t value) {
+  LOGF("OnSimdLaneExpr (lane: %d)\n", value);
```

Comment:
Match function name here: `OnSimd1LaneImmExpr`

Reply:
Sorry for the omission. I will fix it, Thanks!

Reply:
Done.